integration-tests/common/metric.go

```diff
@@ -0,0 +1,48 @@
+package common
+
+import (
+	"encoding/json"
+	"fmt"
+)
+
+type MetricResponse struct {
+	Status string     `json:"status"`
+	Data   MetricData `json:"data"`
+}
+
+type MetricData struct {
+	ResultType string         `json:"resultType"`
+	Result     []MetricResult `json:"result"`
+}
+
+type MetricResult struct {
```

Comment:
Do we need to check the metric type (counter, sum, etc.)? You don't have to do it now, but maybe as a todo